pkg/controller/compliancescan/config.go

```diff
@@ -70,9 +76,10 @@ cmd+=($CONTENT)
 # move the results file when the command is done so the log collector
 # picks up the whole thing and not a partial file
 echo "Running oscap-chroot as ${cmd[@]}"
-"${cmd[@]}"
+"${cmd[@]}" > $REPORT_DIR/cmd_output
 rv=$?
 echo "The scanner returned $rv"
+echo "$rv" > $REPORT_DIR/exit_code
```

Comment:
Are you sure it's safe to add the exit code after the scanner has exited? At this point, we haven't split the ARF and the XCCDF result yet..

Reply:
Right, gotta move this till later then. Good call